automatic/git.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$is64bit = Get-ProcessorBits 64
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\PortableGit-2.11.0-32-bit.7z.exe"
+$filePath64 = "$toolsDir\PortableGit-2.11.0-64-bit.7z.exe"
+$filePath = if ($is64bit -and $env:chocolateyForceX86 -eq $false) { $filePath64 } else { $filePath32 }
```

Comment:
The remaining files problem might not exist. I am not sure what kind of changes choco recently has @ferventcoder . AFAIK old lib directory of the package is cleaned on new install now. I remember that on older choco some stuff would pille up here.